app/assets/javascripts/tests/_map.test.js

```diff
@@ -213,6 +213,21 @@ test('when map was change programatically saveViewport calls params update with
   t.end();
 });
 
+test('removeAllMarkers removes all markers', t => {
+  let markers = [{setMap: (_) => {}}]
+  const service = _map({})
+  service.markers = markers
+  
+  service.removeAllMarkers()
+
+  const expected = [];
```

Comment:
Not sure you want to do the same. I normally keep the expected attached to the expected block

```
 const expected = []
 t.deepEqual(actual, expected)
```